src/devices/tuya.ts

```diff
@@ -5310,6 +5310,165 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        model: 'M9-zigbee-SL',
+        vendor: 'TuYa',
+        description: 'TuYa Smart Switch (4 gang + 4 scene) with neutral wire and motion sensing',
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: '_TZE200_nvodulvi',
+            },
+        ],
+        fromZigbee: [tuya.fz.datapoints, fz.ias_occupancy_only_alarm_2, tuya.fz.indicator_mode],
+        toZigbee: [tuya.tz.datapoints, tuya.tz.power_on_behavior_1, tuya.tz.backlight_indicator_mode_1],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch().withEndpoint('l1'),
+            tuya.exposes.switch().withEndpoint('l2'),
+            tuya.exposes.switch().withEndpoint('l3'),
+            tuya.exposes.switch().withEndpoint('l4'),
+            tuya.exposes.switch().withEndpoint('l5'),
+            tuya.exposes.switch().withEndpoint('l6'),
+            tuya.exposes.backlightModeOffOn().withAccess(ea.STATE_SET),
+            new exposes.Enum('power_on_behavior_l1', ea.STATE_SET, ['off', 'on', 'memory'])
```

Comment:
Please use `e.power_on_behavior().withAccess(ea.STATE_SET)` for all (`memory` will become `previous`)